CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.8.1] - 2022-07-27
```

Comment:
I think we are currently on 3.8.8
https://github.com/codesyncapp/codesync-intellij/blob/master/build.gradle#L9

CHANGELOG wasn't updated with recent pushes.

Reply:
Resolved!